src/Microsoft.ML.CpuMath/Avx.cs

```diff
@@ -73,8 +73,7 @@ public static void MatTimesSrc(bool tran, bool add, AlignedArray mat, AlignedArr
 
             if (iposMin >= iposLim)
             {
-                if (!add)
```

Comment:
Is removing this check safe to do?

Reply:
Ah, I think I get it. No one is actually passing in `true` for this parameter?

---
In reply to: [225329341](https://github.com/dotnet/machinelearning/pull/1218#discussion_r225329341) [](ancestors = 225329341)